app/controllers/reservations_controller.rb

```diff
@@ -138,6 +138,16 @@ def show
     @reservation  = Reservation.find(params[:id])
 
     raise ActiveRecord::RecordNotFound if @reservation != @order_detail.reservation
+
+    respond_to do |format|
+      format.html
+
+      format.ics do
+        calendar = ReservationCalendar.new(@reservation)
+        send_data(calendar.to_ical, type: "text/calendar",
```

Comment:
Part of me still wants to abstract this, but I'd be fine waiting until we have 3 places. 

And not a big deal, but the two controllers we have it in are formatted slightly differently.